modello-plugins/modello-plugin-velocity/src/main/resources/META-INF/plexus/components.xml

```diff
@@ -0,0 +1,31 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<!--
+Licensed to the Apache Software Foundation (ASF) under one
+or more contributor license agreements.  See the NOTICE file
+distributed with this work for additional information
+regarding copyright ownership.  The ASF licenses this file
+to you under the Apache License, Version 2.0 (the
+"License"); you may not use this file except in compliance
+with the License.  You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing,
+software distributed under the License is distributed on an
+"AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+KIND, either express or implied.  See the License for the
+specific language governing permissions and limitations
+under the License.
+-->
+
+<component-set>
+  <components>
```

Comment:
uh, this one should be replaced with annotations

Reply:
This is out of context for this PR.  I think a separate PR that would migrate _all_ modello plugins to annotations would definitely be a good idea, but I'd rather keep them consistent for now.

Reply:
ok, i forgot we did not yet do the move: you're right, let's do it globally later...